app/controllers/marketplaces_controller.rb

```diff
@@ -110,24 +110,24 @@ def update_account
       account = @marketplace.update_account
       if account
         if account.id
-          @activation_message = "Conseguimos com sucesso atualizar sua conta do #{@marketplace.organizer.name}"
+          @activation_message = t("marketplace_controller_activation_message_five")
```

Comment:
Here will be:

`t('marketplace_controller_activation_message_five', organizer: @marketplace.organizer.name) %>`

And on yaml language file file:
`marketplace_controller_activation_message_five: "Conseguimos com sucesso atualizar sua conta do %{organizer}"`

when has variable there's a second parameter in the t function, and on the string you mark with `%{variable}`

So this needs to be fixed in all translations that holds a variable